README.md

```diff
@@ -136,56 +156,85 @@ service<http:Service> order_mgt bind listener {
         // Set the JSON payload to the outgoing response message to the client.
         response.setJsonPayload(existingOrder);
         // Send response to the client.
-        _ = client -> respond(response);
+        _ = client->respond(response);
     }
 
-    @Description {value:"Resource that handles the HTTP DELETE requests, which are
-    directed to the path '/orders/<orderId>' to delete an existing Order."}
+    @Description { value: "Resource that handles the HTTP DELETE requests, which are
+    directed to the path '/orders/<orderId>' to delete an existing Order." }
     @http:ResourceConfig {
-        methods:["DELETE"],
-        path:"/order/{orderId}",
-        authConfig:{
-            scopes:["cancel_order"]
+        methods: ["DELETE"],
+        path: "/order/{orderId}",
+        authConfig: {
+            scopes: ["cancel_order"]
         }
     }
     cancelOrder(endpoint client, http:Request req, string orderId) {
-        http:Response response;
+        // Get untainted string value if `orderId` is a valid input
+        orderId = getUntaintedStringIfValid(orderId);
+
         // Remove the requested order from the map.
         _ = ordersMap.remove(orderId);
 
+        http:Response response;
         json payload = "Order : " + orderId + " removed.";
         // Set a generated payload with order status.
         response.setJsonPayload(payload);
 
         // Send response to the client.
-        _ = client -> respond(response);
+        _ = client->respond(response);
     }
-    
-    @Description {value:"Resource that handles the HTTP GET requests that are directed
-    to a specific order using path '/orders/<orderID>'"}
+
+    @Description { value: "Resource that handles the HTTP GET requests that are directed
+    to a specific order using path '/orders/<orderID>'" }
     @http:ResourceConfig {
-        methods:["GET"],
-        path:"/order/{orderId}",
-        authConfig:{
+        methods: ["GET"],
+        path: "/order/{orderId}",
+        authConfig: {
             authentication: { enabled: false }
         }
     }
     findOrder(endpoint client, http:Request req, string orderId) {
+        // Get untainted string value if `orderId` is a valid input
+        orderId = getUntaintedStringIfValid(orderId);
+
         // Find the requested order from the map and retrieve it in JSON format.
-        json? payload = ordersMap[orderId];
         http:Response response;
-        if (payload == null) {
+        json payload;
+        if (ordersMap.hasKey(orderId)) {
+            payload = ordersMap[orderId];
+        } else {
+            response.statusCode = 404;
```

Comment:
Let's use the constants available in Ballerina for status codes

Reply:
Fixed.